src/OpenLoco/Vehicles/Vehicle.h

```diff
@@ -125,17 +125,17 @@ namespace OpenLoco::Vehicles
         }
 
     public:
-        VehicleThingType getSubType() const { return VehicleThingType(thing_base::getSubType()); }
+        constexpr VehicleThingType getSubType() const { return VehicleThingType(thing_base::getSubType()); }
```

Comment:
revert these constexprs they would never actually be used and just clouds the changes.